bedrock/contentful/tests/test_contentful_api.py

```diff
@@ -1197,44 +1197,116 @@ def test_ContentfulPage__get_info_data(
     )
 
 
-# FURTHER TESTS TO COME
-# def test_ContentfulPage__get_content():
-#     assert False, "WRITE ME"
+@patch("bedrock.contentful.api._get_image_url")
+def test_ContentfulPage__get_split_data(mock__get_image_url, basic_contentful_page):
+    # mock self and entry data
+    basic_contentful_page.page = Mock()
+    basic_contentful_page.page.content_type.id = "mockPage"
+    basic_contentful_page.render_rich_text = Mock()
+    mock_entry_obj = Mock()
+    # only set required and default fields
+    mock_entry_obj.fields.return_value = {
+        "name": "Split Test",
+        "image": "Stub image",
```

Comment:
Better way to mock image and body content?
Less mock boilerplate?

Reply:
hmmm, not sure if there is. Seems like you're only passing the required ones. and Contentful will send a default to the API if none have been set in contentful. maybe @alexgibson or @stevejalim have insight?